<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgPageClass</class>
 <widget class="QWidget" name="MsgPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5,0">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ImgButton" name="msgBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/weiduodianzi/ui/message.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <item>
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelVersionx">
         <property name="text">
          <string>xxxxx</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <item>
        <widget class="QLabel" name="labelLicense">
         <property name="text">
          <string>License:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLicensex">
         <property name="text">
          <string>xxxxx</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
       <item>
        <widget class="QLabel" name="labelSerial">
         <property name="text">
          <string>Serial:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSerialx">
         <property name="text">
          <string>xxxxx</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImgButton</class>
   <extends>QPushButton</extends>
   <header>imgbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="weiduodianzi.qrc"/>
 </resources>
 <connections/>
</ui>
